packages/shared/components/modals/AccountActionsMenu.svelte

```diff
@@ -27,7 +30,11 @@
             type: 'deleteAccount',
             props: {
                 account: selectedAccount,
-                deleteAccount,
+                deleteAccount: async (index: number) => {
+                    await deleteAccount(index)
+                    setSelectedAccount(get(visibleActiveAccounts)?.[0]?.index ?? null)
+                    Platform.resetRouterForAppContext(AppContext.Dashboard)
```

Comment:
This logic should reside inside the original `deleteAccount()` to separate the view from the logic.

Reply:
Yeah, it makes sense now that it's using a `Platform` method.